### ISSUE TITLE
NDK version error

### ISSUE THREAD

**Issue by Mister-Hope** [subosito/flutter-action#54]
```
* What went wrong:

Execution failed for task ':app:stripProfileDebugSymbols'.

> No version of NDK matched the requested version 21.0.6113669. Versions available locally: 21.3.6528147
```

For details see https://github.com/Hope-Studio/inNENU-flutter/runs/887156839?check_suite_focus=true

And 

```
* What went wrong:

Execution failed for task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform libs.jar to match attributes {artifactType=processed-jar, org.gradle.libraryelements=jar, org.gradle.usage=java-runtime}.
      > Execution failed for JetifyTransform: /home/runner/work/inNENU-flutter/inNENU-flutter/build/app/intermediates/flutter/debug/libs.jar.

         > Transform's input file does not exist: /home/runner/work/inNENU-flutter/inNENU-flutter/build/app/intermediates/flutter/debug/libs.jar. (See https://issuetracker.google.com/issues/158753935)
```

For details see https://github.com/Hope-Studio/inNENU-flutter/runs/886996626?check_suite_focus=true

Though I am new to flutter, but I believe the second one is this action's issue, because `build/` folder should not be contained in version control.

For the first one,  I will try to fix it, but I don't think running  `flutter build apk --debug` or  `flutter build apk --profile` is needed to generate `build/app/intermediates/flutter/debug/libs.jar` for a release build.

**Comment by daveleenew** [comment]
+1

**Comment by emme1009** [comment]
same issue here. any updates??

**Comment by cuong0993** [comment]
Got the issue after updating com.android.tools.build:gradle 3.5.2 to 3.6.4. Workaround is adding ndkVersion "21.3.6528147" to android\app\build.gradle.

**Comment by guidezpl** [comment]
The resolution to this is in https://github.com/flutter/flutter/issues/53657#issuecomment-795808016.

TLDR:

Remove `ndkVersion` from build.gradle from and upgrade gradle to 4.1.2 or higher

**Comment by sysarcher** [comment]
Hi Folks... 

I've just started encountering this recently (https://github.com/sysarcher/hack_zurich/runs/3757839044?check_suite_focus=true#step:6:53)

My [Actions file](https://github.com/sysarcher/hack_zurich/blob/main/.github/workflows/main.yml) uses the `stable` channel... Any idea on how I could fix this? I'm stumped!!

**Comment by cuong0993** [comment]
I have time. Just update stuffs.
https://github.com/cuong0993/hack_zurich

**Comment by sysarcher** [comment]
@cuong0993 thanks for the updates!!! But what was the cause? versions mismatch? Why am I able to build it on my machine??

**Comment by cuong0993** [comment]
I am not sure. But:
- The android project is too outdated anyway.
- The github action did not specified the flutter version, may be it is because flutter update to 2.5.1.
